include/control_toolbox/pid.hpp

```diff
@@ -261,7 +354,65 @@ class CONTROL_TOOLBOX_PUBLIC Pid
    *
    * \returns PID command
    */
-  [[nodiscard]] double computeCommand(double error, uint64_t dt);
+  [[deprecated("Use compute_command() instead")]] [[nodiscard]] double computeCommand(
+    double error, uint64_t dt_ns) {
+    return compute_command(error, static_cast<double>(dt_ns) / 1.e9);
+  }
+
+  /*!
+   * \brief Set the PID error and compute the PID command with nonuniform time
+   * step size. The derivative error is computed from the change in the error
+   * and the timestep \c dt.
+   *
+   * \param error  Error since last call (error = target - state)
+   * \param dt Change in time since last call, measured in nanoseconds.
+   *
+   * \returns PID command
+   */
+  [[nodiscard]] double compute_command(double error, rcl_duration_value_t dt_ns) {
+    return compute_command(error, static_cast<double>(dt_ns)/1.e9);
+  }
+
+  /*!
+   * \brief Set the PID error and compute the PID command with nonuniform time
+   * step size. The derivative error is computed from the change in the error
+   * and the timestep \c dt.
+   *
+   * \param error  Error since last call (error = target - state)
+   * \param dt Change in time since last call
+   *
+   * \returns PID command
+   */
+  [[nodiscard]] double compute_command(double error, rclcpp::Duration dt) {
+    return compute_command(error, dt.seconds());
+  }
+
```

Comment:
Can we move all the non-deprecated methods definitions in the .cpp?

Reply:
done